cf_remote/ssh.py

```diff
@@ -37,7 +37,7 @@ def run(self, command, hide=False):
     def put(self, src, hide=False):
         src = os.path.abspath(src)
         dst = os.path.basename(src)
-        if src != dst:
+        if src != dst and not os.path.samefile(src, dst):
```

Comment:
and... this is not a complete solution either if you aren't in the cf-remote source dir.

Will fix.